src/components/pages/career/career-page.tsx

```diff
@@ -47,18 +62,20 @@ export const CareerPage = ({ positions, heroImageData }: CareerPageProps) => {
         </SectionHeader>
       </ContentBlockContainer>
 
+      <ApplicationProcess />
```

Comment:
I would put the `ApllicationProcess` inside the first `ContentBlockContainer`, since the contents belong together (and they are below the same `SectionHeader` on the page)
```suggestion
        </SectionHeader>
        <ApplicationProcess />
      </ContentBlockContainer>

```

Reply:
Hi Felix, thank you for all your suggestions and feedback. 
I have just updated this part and put the `ApllicationProcess` inside the `ContentBlockContainer` from the `SectionHeader`